spinalcordtoolbox/registration/core.py

```diff
@@ -358,6 +362,19 @@ def register(src, dest, step, param):
             verbose=param.verbose,
         )
 
+    elif step.type in ['rootlet']:
+        if step.algo:
+            printv(f"Parameter 'algo={step.algo}' has no effect for 'type=label' registration.", type='warning')
```

Comment:
Ah! I think this copy-pasted warning is not relevant, since technically the user could customize the `algo` for now.

Reply:
now removed in [1e4d052](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4647/commits/1e4d052fe67c54f7cc3b95670ef333803bd639f6)